asdf.fish

```diff
@@ -2,9 +2,9 @@ set -x ASDF_DIR (dirname (status -f))
 
 set -l asdf_user_shims (
   if test -n "$ASDF_DATA_DIR"
-    echo $ASDF_DATA_DIR/shims
+    printf $ASDF_DATA_DIR/shims
```

Comment:
The first argument to `printf` should be the format string. The format string should not contain variables, only placeholders. For example, in this case `printf "%s\n" "$ASDF_DATA_DIR/shims"` to replace `echo`.

Reply:
Changed in e697aaba1d15ea53efbddb31b382a7df0de379bb